src/common.js

```diff
@@ -56,18 +56,11 @@ async function runKeyguard(RequestApiClass, options) { // eslint-disable-line no
     /** @type {TopLevelApi | IFrameApi} */
     const api = new RequestApiClass();
 
-    /**
-     * @returns {string}
-     */
-    function allowedOrigin() {
-        switch (window.location.origin) {
-        case 'https://keyguard-next.nimiq.com': return 'https://accounts.nimiq.com';
-        case 'https://keyguard-next.nimiq-testnet.com': return 'https://accounts.nimiq-testnet.com';
-        default: return '*';
-        }
-    }
+    /** @type {string} */
+    // eslint-disable-next-line no-undef
+    const allowedOrigin = CONFIG_ALLOWED_ORIGIN || '*';
```

Comment:
I think our accounts manager should be the default here, because the keyguard is a sensible little thing with all the keys in it etc. so I think making `*` the default is not good. The default should already be safe.

Wdyt, @danimoh @nibhar?

Reply:
I think testnet. Mainnet should require a build flag, just like production build usually require a build flag.

Reply:
Problem is, however, that it should default to '*' for unbuilt versions, shouldn't it?

Reply:
Happy now? At first I wanted to replace the '*' in source code, but then we would loose our ability to use that as an entry point. If you don't think it's enough, we could reintroduce the check of own origin in src.

Reply:
Ok, so for locally running it without a build it now uses `*`, but the build script uses the testnet config as the default. I think this is ok now.